index.html

```diff
@@ -188,25 +194,173 @@ <h2 class="pr-title">
+                <li class="pr-tag">javascript</li>
+            </ul>
+            <!--action button -->
+            <div class="pr-call-to-action-box">
+                <a href="#" class="pr-call-to-action-button"> See Project</a>
+            </div>
+        </div>
+    </div>
+    <!-- fourth card-->
+    <div class="card-four">
+        <!--portfolio image four -->
+        <div class="pr-photo4">
+            <img class="portfolio-img" src="./images/photos/portfolio-pic-3.png" alt="4th portfolio photo">
+        </div>
+        <!--portfolio details one -->
+       <div class="pr-details4"> 
+            <!--project title-->
+            <h2 class="pr-title">
+                Uber Navigation
+            </h2>
+            <!--project type-->
+            <ul class="pr-type">
+                <li class="no-style"> UBER </li>
+                <li> Leader Developer </li>
+                <li> 2018 </li>
+            </ul>
+            <!--project description-->
+            <p class="pr-description ">
+                A smart assistant to make driving more safe, 
+                efficient, and fun by unlocking your most expensive computer: your car.
+            </p>
+            <!--tech tags-->
+            <ul class="pr-tech-tags no-style">
+                <li class="pr-tag">html</li>
+                <li class="pr-tag">Ruby on rails</li>
+                <li class="pr-tag">css</li>
+                <li class="pr-tag"> javascript</li>
+            </ul>
+            <!--action button -->
+            <div class="pr-call-to-action-box">
+                <a href="#" class="pr-call-to-action-button"> See Project</a>
+            </div>
+        </div>
+    </div>
+
+</section> 
+
 <!--   --About-me section-->
 <section class="about-me-section">
     <!--Top begins here - Text intro social connect , social icons and action button-->
 
             <div class="about-me-top">
                     <h2 class="primary-text abt-primary-text">
-                        About <br> Myself
+                        About <br class="primary-text-br"> Myself
                     </h2> 
-                    <p class="intro-text">
+                    <p class="intro-text abt-intro-text">
                         Hello, I&#39;m a software developer! <br> I can help you build a software product, feature or website. 
                         Take a look at some of my works and experience! If you like what you see and 
                         have a project, don&#39;t hestiate to contact me.
                     </p> 
                     <!--social media -->
-                    <p class="social-media-text">
+                    <p class="social-media-text abt-social-media-text">
                         LET&#39;S CONNECT 
                     </p>
                     <!--social media icons-->
-                    <ul class="social-media-icons">
+                    <ul class="social-media-icons abt-social-media-icons">
```

Comment:
- [ ] Good job making everything done 👍 , kindly make sure you correct the text in this button is well aligned 
![Capture 2](https://user-images.githubusercontent.com/80458199/230695398-70ba31d4-0b4a-49f2-be02-ce75e7429972.PNG)

Reply:
Thanks. I will check it. It looked fine locally though.

Reply:
It was difficult to see the error, but has refactored the styles to effect any such correction. Thanks